Yoast/wordpress-seo
# Implement slug data flow

## Summary

This PR can be summarized in the following changelog entry:

* Make sure slug is the same in gutenberg, seo analysis and in the snippet preview.

## Relevant technical choices:

* This is not an ideal solution, because now we are having to keep the slug in sync in both the gutenberg store and in our own store. This will be fixed when we start using `withSelect` and `withDispatch` in out won components, so we can get the slug from the gutenberg store.

## Test instructions

This PR can be tested by following these steps:

For both Gutenberg and the classic editor:

* Update the slug in the editor, make sure it is updated in the snippet editor and in the analysis.
* Update the slug in the snippet editor, make sure it is updated in the editor and in the analysis.

## Quality assurance


Fixes #10499
